tests/Responses/Expected/taxon/missing_locale_exception_response.json

```
@@ -0,0 +1,4 @@
+{
+    "code": 400,
+    "message": "You haven't provide a locale!"
```

A bit of grammar change: "`You did not provide a locale!"`

```suggestion
    "message": "You did not provide a locale!"
```